### PR TITLE
Vacation api integration

### LINKED ISSUE #20: Userstory: Urlaubs view JS
...

umsetzung:
JS funktionen schreiben die mit dem backend interagiert und mit den anderen funktionen die urlaubs ansicht seite fuellt.

### LINKED ISSUE #32: Userstory: Vacation APIs Integration
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

### PR DESCRIPTION
**Pull request by dikkadev**
fixes #20 
fixes #32

### CODE REVIEW

**Comment by dikkadev** [comment]
test and merge when real user id is available from login

**Comment by dikkadev** [comment]
PS: user in vacation backend is not created when registering. BUUUUT thats not our job

File: js/urlaubsantrag-stellen.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
   let data = {
-    userId: DBGID,
+    userId: user.id,
     startDate: new Date(form.querySelector('input[name="von"]').value).getTime(),
     endDate: new Date(form.querySelector('input[name="bis"]').value).getTime(),
-    comment: form.querySelector('textarea[name="comment"]').value,
     status: 'beantragt',
   }
 
   fetch('https://studium.webfajo.de/mitarbeiter/antragstellung', {
```
Comment (Sett17) [inline js/urlaubsantrag-stellen.js:13]: ATTOW timeout no matter which id or auth

File: js/urlaubsantrag-stellen.js [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+    startDate: new Date(form.querySelector('input[name="von"]').value).getTime(),
+    endDate: new Date(form.querySelector('input[name="bis"]').value).getTime(),
+    status: 'beantragt',
+  }
 
-    if (isGleitzeitChecked) {
-        // Checkbox is set to Gleitzeit
-        inputUrlaubstage.readOnly = false;
-        inputUrlaubstage.setAttribute('style', "width: 3em");
+  fetch('https://studium.webfajo.de/mitarbeiter/antragstellung', {
```
Comment (dikkadev) [inline js/urlaubsantrag-stellen.js:13]: ATTOW timeout no matter id or auth

Comment (dikkadev) [reply js/urlaubsantrag-stellen.js:13]: fixed

**Pull request merged**